parsec/core/fs/workspacefs/entry_transactions.py

```diff
@@ -180,6 +181,35 @@ async def _lock_parent_manifest_from_path(
             await self._load_manifest(entry_id)
 
     # Transactions
+    async def entry_missing_data(
+        self, path: FsPath, limit: int
+    ) -> Tuple[int, int, List[BlockAccess]]:
+        missing_blocks = []
+        manifest, confinement_point = await self._get_manifest_from_path(path)
+        manifest: LocalFileManifest
+        accessible_chunks = []
```

Comment:
`accessible_chunks` (in the sense of `chunks with an access`) should be called `blocks`